web/src/main/java/se/inera/intyg/webcert/web/web/controller/integration/IntygIntegrationController.java

```diff
@@ -498,5 +512,22 @@ private void validateRequest(Map<String, Object> pathParameters) {
         super.validateParameters(pathParameters);
         super.validateAuthorities();
     }
+
+    private String assertLaunchIdFormat(String launchId) {
+        if (launchId.isEmpty()) {
+            return null;
+        }
+        if (launchId != null) {
```

Comment:
Redundant check. The check above should take care of the validation and return the method if its not relevant to continue.

Reply:
Gör validering innan vi sätter parametern!